backend/app/api/endpoints/attachments_open.py

```
@@ -0,0 +1,196 @@
+
+    return AttachmentResponse.from_context(context, response_truncation_info)
+
+
+@router.post(
+    "/upload",
+    response_model=AttachmentResponse,
+    status_code=status.HTTP_201_CREATED,
+)
+@trace_async("upload_attachment_open", "attachments.api")
+async def upload_attachment_open(
+    file: UploadFile = File(...),
```

Ruff (0.15.10)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)